app/assets/stylesheets/custom.scss

```diff
@@ -20,3 +20,52 @@ abbr[title="required"] {
 .panel .panel-footer {
   margin-bottom: 0;
 }
+
+/* Devise flash */
+
+.alert-error {
```

Comment:
Devise uses [:notice, :error, :alert] and you can't change or override them.  These styles format them to match the bootstrap styles [:success, :info, :warning, :danger]